src/js/Controllers/DisplayCapabilities.js

```diff
@@ -603,6 +603,25 @@ let capabilities = {
 				scale: 1,
 				preferredFPS: 30,
 				additionalVideoStreamingCapabilities: [
+					{
```

Comment:
My understanding of the conversation with mobile teams was that this was not necessary. Mobile will parse the top level.

Reply:
I agree, removed duplicate capability in https://github.com/smartdevicelink/generic_hmi/pull/345/commits/82eb1a16f2739e5e2bccd7755c02278356cab691